tests/components/teslemetry/test_sensor.py

```diff
@@ -25,3 +30,30 @@ async def test_sensors(
     entry = await setup_platform(hass, [Platform.SENSOR])
 
     assert_entities(hass, entry.entry_id, entity_registry, snapshot)
+
+    # Coordinator refresh
```

Comment:
Before moving time can you change the mock so the entity state should change once coordinator has made its refresh. 
That way we know coordinator update runs as it's supposed to.

Reply:
So instead of doing two tests (primary and alt) I will instead just use the alt data in the coordinator refresh, achieving both things with 1 test.

Reply:
I don't know what's the difference between primary and alt (checked some but didn't find any difference).

But I think this is fine and also now we can see that the coordinator updates fine on sync-interval.